devel/py-pip/Makefile

```diff
@@ -26,3 +26,10 @@ do-test:
 .include "../../lang/python/application.mk"
 .include "../../lang/python/egg.mk"
 .include "../../mk/bsd.pkg.mk"
+
+.if !empty(_PYTHON_VERSION:M2*)
+ALTERNATIVES_SRC=  ${PKGDIR}/ALTERNATIVES.python2
+.endif
+.if !empty(_PYTHON_VERSION:M3*)
+ALTERNATIVES_SRC=  ${PKGDIR}/ALTERNATIVES.python3
+.endif
```

Comment:
Normally in package Makefile `bsd.pkg.mk` inclusion is the last line.
Please move that logic before `bsd.pkg.mk` inclusion, betweet the REPLACE_PYTHON paragraph and post-install, and add:

````
.include "../../lang/python/pyversion.mk"
[... logic to select ALTERNATIVES_SRC ...]
````